develop/jsapi/work_with_proxy/lab.md

```diff
@@ -0,0 +1,15 @@
+### Working with proxy
+
+This lab covers the basics for allowing public, _no-login-required_ access to secure ArcGIS Online services using a service proxy hosted on ArcGIS Online.
+
+1. Click [work_with_proxy/index.html](../work_with_proxy/index.html) and copy the contents to a new [jsbin.com](http://jsbin.com).
+
+2. Go to your ArcGIS Developers account, select [Dashboard](https://developers.arcgis.com/dashboard) and then select [Create New App](https://developers.arcgis.com/applications/#/new/).
+
+3. Give your new application a title, fill in the appropriate tags and give it a description. When you are done click `Register New Application`.
+
+4. Within the applications page select the `Authentication` tab at the top. Scroll down until you see the section titled `Configure New Service Proxy`.
+
+5. Using the pulldown menu under `Service` select `route`. Then set `Request Limit` to `3` per second. Then click `Configure`.
+
+6. Copy the `Proxy Url` and replace the `RouteTask.url` string in your jsbin app. The app should now be ready to use.
```

Comment:
folks get tripped up on what it means to 'Register a New Application', so perhaps we could articulate something like this:

_The application we are creating doesn't directly reference any of our code, but rather gives us the opportunity to designate a unique url to make requests to a premium service without passing a token.  This ensures that the credit cost of routes calculated will be the responsibility of the account used to create the proxied service._